ressurshefte/ressurshefte.md

```diff
@@ -20,15 +20,19 @@ Med disse byggeklossene kan vi lage rettede grafer som består av ting og relasj
 
 ## Modellere Ada Lovelace
 
-AdaLovelace type Person
+Vi kan modellere personopplysninegen til en fiktiv person Ada Lovelace på følgende måte:
 
-AdaLovelace navn Ada Lovelace
+navn: Ada Lovelace  
+fødselsdato: 10.12.1815  
+bekjente: Charles Babbage  
+interesse: Programmering  
 
-AdaLovelace født 10.12.1815
-
-AdaLovelace kjenner CharlesBabbage
-
-AdaLovelace interesse Programmering
+Vi formulerer dette til en graf (subjekt, predikat, objekt):  
+AdaLovelace har type Person  
+AdaLovelace har navn "Ada Lovelace"  
+AdaLovelace har fødselsdato 10.12.1815  
+AdaLovelace kjenner Charles Babbage  
+AdaLovelace har interesse Programmering  
```

Comment:
Foreslår å beholde ettords-egenskapene ("type", "navn", "interesse", "kjenner" , osv.) for å tydeliggjøre hva som er subjekt, predikat og objekt i denne uformelle syntaksen.